conda/cli/main_config.py

```diff
@@ -346,9 +353,121 @@ def print_config_item(key, value):
                 stdout_write(" ".join(("--add", key, repr(item))))
 
 
-def execute_config(args, parser):
+def _get_key(
```

Comment:
I think `_get_key` is a little confusing for a function name. Usually `get*` functions/methods return something. This just prints some output.

Would `_show_key` be a better name for this?

Reply:
Another thing that might this function easier to reason about: it's really just trying to figure out the correct value to use for `sub_config`. Couldn't we just return this as a value?

I just find this function confusing because the ultimate outcome is either mutating the  `json` dictionary or printing something. It would be nice to rip that last little piece of logic out and further restrict what this function is responsible for.

Reply:
> Usually `get*` functions/methods return something.

No disagreement there, the helper functions were named based on what CLI arguments trigged the code path/where they are called within `execute`. I do not necessarily agree with their implementations.

The goal of this PR was only to remove usage of deprecated functionality (https://github.com/conda/conda/pull/13886#discussion_r1589739364). Additional refactoring for this is desired, necessary, and expected after the 24.5.x release.